extensions/wikia/WikiFactory/Close/templates/close-info.tmpl.php

```diff
@@ -26,7 +26,7 @@
 <table width="90%" align="center" id="closed-urls">
     <tr>
         <td width="50%" style="text-align:center"><a href="http://www.wikia.com/Special:CreateWiki"><?= wfMessage( 'closed-wiki-create-wiki' )->parse() ?></a></td>
-        <td width="50%" style="text-align:center"><a href="/wiki/Wikia:Closed_Wikia"><?= wfMessage( 'closed-wiki-policy' )->escaped() ?></a></td>
+        <td width="50%" style="text-align:center"><a href="<?= wfMessage( 'closed-wiki-policy-url' )->escaped() ?>"><?= wfMessage( 'closed-wiki-policy' )->escaped() ?></a></td>
```

Comment:
Since it's setting the URL, as a best practice it is good to use `->useDatabase( false )` so it can't be overridden locally (though not as problematic now that local messages can't be edited unless whitelisted).